.docker/entrypoint.sh

```diff
@@ -23,7 +23,7 @@ alias delib_ws='cd $ROS_WS'
 function delib_build() {
         cwd=$(pwd)
         delib_ws
-        colcon build --symlink-install --continue-on-error
+        colcon build --symlink-install --continue-on-error --mixin compile-commands
```

Comment:
If we're including this mixin change, include it in the other bash functions that also call `colcon build`

Reply:
ok